Dockerfile.pi

```diff
@@ -0,0 +1,41 @@
+FROM balenalib/raspberrypi3-debian:stretch
+
+RUN [ "cross-build-start" ]
+
+RUN apt-get update \
+ && apt-get -y install wget curl build-essential libssl-dev libffi-dev \
+ && apt-get clean
+
+# Prepare environment
+RUN mkdir /freqtrade
+WORKDIR /freqtrade
+
+# Install TA-lib
+COPY build_helpers/ta-lib-0.4.0-src.tar.gz /freqtrade/
+RUN tar -xzf /freqtrade/ta-lib-0.4.0-src.tar.gz \
+ && cd /freqtrade/ta-lib/ \
+ && ./configure \
+ && make \
+ && make install \
+ && rm /freqtrade/ta-lib-0.4.0-src.tar.gz
+
+ENV LD_LIBRARY_PATH /usr/local/lib
+
+# Install berryconda
+RUN wget https://github.com/jjhelmus/berryconda/releases/download/v2.0.0/Berryconda3-2.0.0-Linux-armv7l.sh \
+ && bash ./Berryconda3-2.0.0-Linux-armv7l.sh -b \
+ && rm Berryconda3-2.0.0-Linux-armv7l.sh
+
+# Install dependencies
+COPY requirements.txt /freqtrade/
+RUN sed -i -e '/^numpy==/d' -e '/^pandas==/d' -e '/^scipy==/d' ./requirements.txt \
```

Comment:
Can we create a `requirements_pi.txt` which does not list these requirements?

That would be the easier (and cleaner) solution as it'll work for people not running on docker too -  and if it's aded to `.pyup.yml` they are updated automatically as well.

Reply:
@xmatthias I didn't know about .pyup.yml, that's why :D. I will add it back,  thanks!